src/Book/Book.tsx

```diff
@@ -42,10 +42,27 @@ type Props = {
 }
 
 const Book = (props: Props) => {
+
+    const BAD_REQUEST = 400;
+    const UNAUTHORIZED = 401;
+    const TOO_MANY_REQUESTS = 429;
+    const EXPECTATION_FAILED = 417;
+    const NOT_ACCEPTABLE = 406;
+
     const [status, setStatus] = React.useState(SEARCH);
 
     const [selectedBook, setSelectedBook] = React.useState<Book | undefined>(undefined)
 
+    const toastifyConfigurations : ToastOptions = {
+        position: "top-center",
+        autoClose: 7000,
+        hideProgressBar: true,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined
+    }
```

Comment:
Is not necessary to set this configuration on every screen. You should set the default configurations on the `<ToastContainer/>` and only use the configuration option if, for some reason, the alert on the current screen should render differently from the default.